_harp/agency/index.ejs

```diff
@@ -1,11 +1,12 @@
 <div class="academy-landing">
 	<h1 class="margin agency">We Know JavaScript.</h1>
+	<a href="https://docs.google.com/forms/d/1u5f_efLMX9dFQSAMnNv1Wx54CDC7XbU9QMpfCRntFNw/viewform" class="margin" target="_newtab"><button class="button-three">WORK WITH US</button></a>
 </div>
 <div class="container">
 
 	<div class="marketing-section">
 		<div>
-			<h1>The Founders & Coders agency is made up of over 30 freelancers. We specialise in building web and mobile applications in JavaScript and Node.js. Our developers have worked with startups, Government and FTSE 100 companies so chances are we can work with you.</h1>
+			<h1>Founders & Coders is home to over 30 freelancers. We specialise in building web and mobile applications in JavaScript and Node.js. Our developers have worked with startups, Government and FTSE 100 companies so chances are we can work with you.</h1>
```

Comment:
'They' instead of 'our developers' and 'they' instead of 'we' (give the plan is to separate F&C academy from the freelance work)